### ISSUE TITLE
Exception while running integration tests using SunnyDay profile

### ISSUE THREAD

**Issue by mikewalch** [apache/accumulo#411]
Exception occurred in monitor module:
```
java.lang.RuntimeException: Unable to load category: org.apache.accumulo.test.categories.SunnyDayTests
	at org.apache.maven.surefire.group.match.SingleGroupMatcher.loadGroupClasses(SingleGroupMatcher.java:142)
	at org.apache.maven.surefire.common.junit48.FilterFactory.createGroupFilter(FilterFactory.java:100)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.createJUnit48Filter(JUnitCoreProvider.java:279)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:373)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:334)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:407)
Caused by: java.lang.ClassNotFoundException: org.apache.accumulo.test.categories.SunnyDayTests
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at org.apache.maven.surefire.group.match.SingleGroupMatcher.loadGroupClasses(SingleGroupMatcher.java:138)
	... 7 more

```

**Comment by ctubbsii** [comment]
I've reproduced the problem, and am working on a fix.

**Comment by ctubbsii** [comment]
I actually think this is a bug in the maven-surefire-plugin / maven-failsafe-plugin code. I will push a fix here, but have also sent a note to the Maven dev list.

**Comment by ctubbsii** [comment]
Response from the Maven list indicated this was fixed in a newer version of surefire/failsafe. I'm going to revert my previous change, in favor of bumping the surefire/failsafe plugin to the latest.

**Comment by ctubbsii** [comment]
Bumping to the newer version results in errors about "Corrupted STDOUT". I'll leave the existing fix in place for now, rather than revert.

**Comment by ctubbsii** [comment]
The Corrupted STDOUT issue should be fixed in 3.0.0-M1 (2.20.1 is our current version, and 2.21.0 is the version that fixes the problem, but creates the corrupted STDOUT)

https://issues.apache.org/jira/browse/SUREFIRE-1498
